spec/support/features/in_person_helper.rb

```diff
@@ -8,6 +8,9 @@ module InPersonHelper
   GOOD_FIRST_NAME = Idp::Constants::MOCK_IDV_APPLICANT[:first_name]
   GOOD_LAST_NAME = Idp::Constants::MOCK_IDV_APPLICANT[:last_name]
   GOOD_DOB = Idp::Constants::MOCK_IDV_APPLICANT[:dob]
```

Comment:
I think we can delete this line, it doesn't look like it's used anywhere else besides the specs updated in this pr